3rdparty/python/requirements.txt

```diff
@@ -17,8 +15,8 @@ psutil==5.4.8
 Pygments==2.3.1
 pyopenssl==17.3.0
 pystache==0.5.3
-pytest-cov>=2.5,<2.6
-pytest>=3.4,<4.0
+pytest-cov>=2.6,<2.8
+pytest>=4.1,<5.0
```

Comment:
I think we'll need this to be 4.2.1+ to ensure `more-itertools` isn't a conflict.

Also, would be good to add a comment `We pin this <5 to ensure that Python 2 still works.`